client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import { Alert, Button, FormGroup, Spinner } from 'reactstrap';
+import { FormField } from '_common';
+import * as Yup from 'yup';
+import './FeedbackForm.module.scss';
+
+const formSchema = Yup.object().shape({
+  subject: Yup.string().required('Required'),
+  name: Yup.string().required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  problem_description: Yup.string().required('Required')
+});
+
+const FeedbackForm = ({ authenticatedUser }) => {
+  const dispatch = useDispatch();
+  const creating = useSelector(state => state.ticketCreate.creating);
+  const creatingError = useSelector(state => state.ticketCreate.creatingError);
+  const creatingErrorMessage = useSelector(
+    state => state.ticketCreate.creatingErrorMessage
+  );
+
+  const defaultValues = useMemo(
+    () => ({
+      subject: 'Feedback for the Frontera Portal',
+      problem_description: '',
+      first_name: authenticatedUser ? authenticatedUser.first_name : '',
+      last_name: authenticatedUser ? authenticatedUser.last_name : '',
+      name: authenticatedUser
+        ? `${authenticatedUser.first_name} ${authenticatedUser.last_name}`
+        : '',
+      email: authenticatedUser ? authenticatedUser.email : ''
```

Comment:
I am not getting `name` or `email` to populate my modal
<img width="617" alt="Screen Shot 2020-11-10 at 3 13 27 PM" src="https://user-images.githubusercontent.com/20326896/98734323-6a7b4800-2367-11eb-87cc-164e5a09d603.png">

Reply:
I got that once, my session was expired. Is that the case?

Reply:
Works now!